phpunit/functional/PlanningTest.php

```diff
@@ -325,4 +325,133 @@ public function testAllDayEvents()
         $this->assertEquals(__FUNCTION__ . '_not_allday', $events[2]['title']);
         $this->assertFalse($events[2]['allDay'] ?? false);
     }
+
+    public function testCheckAlreadyPlannedProvider()
+    {
+        $tech_id    = getItemByTypeName('User', 'tech', true);
+        $normal_id  = getItemByTypeName('User', 'normal', true);
+        $glpi_id    = getItemByTypeName('User', 'glpi', true);
+
+        $begin_task = '2025-05-13 00:00:00';
+        $end_task   = '2025-05-13 01:00:00';
+        $ticket_name = 'Ticket Test planned';
+
+        $ticket_id = $this->createItem(\Ticket::class, [
```

Comment:
phpunit data providers must not create items in databse. I guess this is why tests are failing.